IonDotnet/IIonDecimal.cs

```diff
@@ -1,7 +1,10 @@
 ﻿namespace IonDotnet
 {
+    // An Ion Decimal Value
     public interface IIonDecimal : IIonValue<IIonDecimal>
     {
-        decimal DecimalValue { get; set; }
+        decimal DecimalValue { get; set; }  
+        float FloatValue { get; set; }
```

Comment:
We might consider remove `FloatValue` property here. `DoubleValue` should be enough.